webpack.config.js

```diff
@@ -256,6 +256,11 @@ module.exports = () => {
               filename: SMOKE_PREFIX + plugin.options.filename
             })
           );
+        } else if (plugin instanceof LodashTemplatePlugin) {
+          return new LodashTemplatePlugin({
+            ...plugin.options,
+            imports: { ...plugin.options.imports, HTML_PREFIX: "smoke-" }
```

Comment:
should this use [the `SMOKE_PREFIX` constant](https://github.com/mozilla/hubs/blob/763680/webpack.config.js#L13)?

Reply:
yes, nice catch!